### PR TITLE
#305723897 - Venue section

### PR DESCRIPTION
**Pull request by dmatis2**
Monday.com: https://hackkosice.monday.com/boards/227660128/pulses/305723897

### CODE REVIEW

File: src/organisms/home/Venue.vue [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+      <MapViewer
+        :position="{ lat: 48.72483, lon: 21.261012 }"
+        :zoom="17"
+        :enablePicker="true"
+        :fixed="false"
+        :tooltip="{
+          title: 'Kunsthalle Košice',
+          description: 'Rumanova 1, 040 01 Košice, Slovakia'
+        }"
+        style="height: 500px"
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: Move to CSS

File: src/organisms/home/Venue.vue [hunk 2]
```diff
@@ -1,7 +1,71 @@
...
+            alt="🇸🇰"
+            src="https://s.w.org/images/core/emoji/11/svg/1f1f8-1f1f0.svg"
+          /><b>Kosice, Slovakia</b></SubPageHeading
+        >
+      </div>
+      <SubPageHeading>
+        <b>Venue:</b> Kunsthalle/Hala umenia Košice
+      </SubPageHeading>
+      <MapViewer
+        :position="{ lat: 48.72483, lon: 21.261012 }"
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: Might be better if you move complex objects to computed properties, what do you think?

File: src/organisms/home/Venue.vue [hunk 3]
```diff
@@ -1,7 +1,71 @@
 <template>
-  <div>
+  <div id="venue-background">
     <div class="container">
-      Venue
+      <PageHeading :display="5">Location</PageHeading>
+      <div class="location">
+        <SubPageHeading
+          ><img
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: Fix ESLint's formatting

File: src/organisms/home/Venue.vue [hunk 4]
```diff
@@ -1,7 +1,71 @@
...
+    SubPageHeading,
+    MapViewer
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+#venue-background {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)),
+    url("https://media.kosiceregion.com/assets/50196/large_hala_umenia_4.jpg");
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: This is not wise, if they ever change that asset, it will break our page. Definitely download it locally.

File: src/pages/Home.vue [hunk 5]
```diff
@@ -3,8 +3,8 @@
     <WelcomeSection />
     <WhatIsHK />
     <PreRegistration />
-    <SponsorsPartners />
```
Comment (MatusMak) [inline src/pages/Home.vue]: Revert this change

File: src/pages/policy/PrivacyPolicy.vue [hunk 6]
```diff
@@ -46,9 +46,9 @@
       />.
     </TextParagraph>
 
-    <TextHeading class="mt-5">
-      Accessing, changing and removing your information
-    </TextHeading>
+    <TextHeading class="mt-5"
+      >Accessing, changing and removing your information</TextHeading
```
Comment (MatusMak) [inline src/pages/policy/PrivacyPolicy.vue]: Again, fix ESLint's formatting

**Review by MatusMak — CHANGES_REQUESTED** [review]

File: src/organisms/home/Venue.vue (hunk 1 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

File: src/organisms/home/Venue.vue (hunk 2 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: done

File: src/organisms/home/Venue.vue (hunk 3 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

File: src/organisms/home/Venue.vue (hunk 4 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

File: src/pages/Home.vue (hunk 5 above)
Comment (dmatis2) [reply src/pages/Home.vue]: Sponsors & Partners section is below Venue section

File: src/pages/policy/PrivacyPolicy.vue (hunk 6 above)
Comment (dmatis2) [reply src/pages/policy/PrivacyPolicy.vue]: Done

File: src/pages/Home.vue (hunk 5 above)
Comment (dmatis2) [reply src/pages/Home.vue]: But done

File: src/organisms/home/Venue.vue [hunk 7]
```diff
@@ -1,7 +1,79 @@
...
+      <SubPageHeading>
+        <b>Venue:</b> Kunsthalle/Hala umenia Košice
+      </SubPageHeading>
+      <MapViewer
+        :position="getPosition"
+        :zoom="17"
+        :enablePicker="true"
+        :fixed="false"
+        :tooltip="{
+          title: 'Kunsthalle Košice',
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: Same as `getPosition`.

File: src/organisms/home/Venue.vue [hunk 8]
```diff
@@ -1,7 +1,79 @@
...
+  name: "VenueSection",
+
+  components: {
+    PageHeading,
+    SubPageHeading,
+    MapViewer
+  },
+
+  computed: {
+    getPosition: function() {
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: I would treat it as a variable name, not method name.

File: src/organisms/home/Venue.vue [hunk 9]
```diff
@@ -1,7 +1,79 @@
 <template>
-  <div>
+  <div id="venue-background">
     <div class="container">
-      Venue
+      <PageHeading :display="5">Location</PageHeading>
+      <div class="location">
+        <SubPageHeading><img
+            alt="🇸🇰"
+            src="https://s.w.org/images/core/emoji/11/svg/1f1f8-1f1f0.svg"
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: Download this locally as well

File: src/organisms/home/Venue.vue [hunk 10]
```diff
@@ -1,7 +1,79 @@
 <template>
-  <div>
+  <div id="venue-background">
     <div class="container">
-      Venue
+      <PageHeading :display="5">Location</PageHeading>
+      <div class="location">
+        <SubPageHeading><img
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue:6]: Fix ESlint error

**Review by MatusMak — CHANGES_REQUESTED** [review]

File: src/organisms/home/Venue.vue [hunk 11]
```diff
@@ -1,7 +1,79 @@
...
+            alt="🇸🇰"
+            src="https://s.w.org/images/core/emoji/11/svg/1f1f8-1f1f0.svg"
+          /><b>Kosice, Slovakia</b></SubPageHeading>
```
Comment (MatusMak) [inline src/organisms/home/Venue.vue]: End tag should be on a new line

**Review by MatusMak — CHANGES_REQUESTED** [review]

File: src/organisms/home/Venue.vue (hunk 7 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

File: src/organisms/home/Venue.vue (hunk 9 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: done

File: src/organisms/home/Venue.vue (hunk 11 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

File: src/organisms/home/Venue.vue (hunk 8 above)
Comment (dmatis2) [reply src/organisms/home/Venue.vue]: Done

**Review by MatusMak — APPROVED** [review]
